modules/apps/frontend-js/frontend-js-clay-sample-web/src/main/java/com/liferay/frontend/js/clay/sample/web/internal/portlet/FrontendJSClaySampleWebPortlet.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.clay.sample.web.internal.portlet;
+
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCPortlet;
+
+import javax.portlet.Portlet;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Bryce Osterhaus
+ */
+@Component(
+	immediate = true,
+	property = {
+		"com.liferay.portlet.display-category=category.sample",
+		"com.liferay.portlet.instanceable=true",
```

Comment:
Not sure it makes sense to make this `instanceable`, since we're not taking any view-specific params or config, so basically it won't matter how many instances of this we place in a page, they will all show the same.

Reply:
tbh, I don't even know what this property does, just copy pasta from another module. I can remove it